### DISCUSSION TITLE
What determines transport - "Auto"?

### DISCUSSION THREAD

**Discussion by alexbot89** [microsoft/azure-container-apps#627] [Q&A]
I'm looking at the Azure Portal on one of my container apps and the transport is set to Auto with Http/1 and Http/2, could not find any documentation on what "auto" means and how it determines which protocol version to use.

![image](https://user-images.githubusercontent.com/86128101/218650661-0441324e-7676-4be4-abe9-d833069751b8.png)

**Comment by v-vish** [comment]
@alexbot89 When set to Auto, Azure Container Apps will try to negotiate the best protocol between HTTP/1 and HTTP/2 based on the client's request. This means:

If the client supports HTTP/2, it will be used.
If the client does not support HTTP/2, the request falls back to HTTP/1.
